### DISCUSSION TITLE
Question about the Contributing guidelines

### DISCUSSION THREAD

**Discussion by rupsis** [RayTracing/raytracing.github.io#1346] [Q&A]
In the Contributing guidelines, under `Pull Request`:

1. (4 `Create your changes in a feature branch (or fork)`. When creating a fork (The typical way I've contributed before), there's no way to add the "rt-contributors" as reviewers, as you'd need to make them collaborators of the project. Is the preferred way to make a PR directly against this repo?
2. (6 `Include a one-line summary change at the bottom of the current development section in the changelog.`  Is this still the case? I assume it's against the `CHANGELOG.md` but i haven't seen any evidence of other PR's making changes to that.

**Comment by hollasch** [comment]
Good grief — someone who actually read the `CONTRIBUTING.md` doc! Where are you from? Are there more like you? :smile:

There's a better way to accomplish this in Markdeep, so I've delimited all free-standing URLs in the three books, and posted a separate review. Feel free to comment there (linked to from your original review): https://github.com/RayTracing/raytracing.github.io/pull/1347

In the future, the easiest route is to just make the changes directly off the main repository. You might not have permissions for this, so let me know and I can add you as a contributor.

**Reply by rupsis** [reply]
I sure hope there are more like me! 

I'll go ahead and make change directionally in this repo, so If you can give me permissions, that would be great. I look forward to helping out wherever I can!

**Reply by rupsis** [reply]
@hollasch Even with contributor permissions, doesn't look like there's a `rt-contributors` option for requesting a review (which means the README is wrong, eek!). A potential solution could be [code-owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).
